extension-manifest-v3/src/pages/panel/views/home.js

```diff
@@ -242,5 +290,22 @@ export default {
         </section>
       `}
     </template>
+  `.css`
```

Comment:
Don't create CSS in views. It should be only in components. If you need to change visuals, change it in the corresponding component, or make a new one.